### DISCUSSION TITLE
Who is using APIFlask?

### DISCUSSION THREAD

**Discussion by greyli** [apiflask/apiflask#273] [Meta]
If you use APIFlask in your side project or in your company, let us know! Just leave your website/project link/company name/feedback here.

Example:

```
- Organization:
- Location: 
- Project name/description: 
- Project link:

... (any feedback will be helpful!)
```

**Comment by andyzhouty** [comment]
- Organization: [Floggers](https://github.com/flog-team)
- Project name & link: [Flog](https://github.com/flog-team/flog-api-v4)

**Comment by greyli** [comment]
- Organization: Dell
- Location: Shanghai, China
- Project name/description: OSS Doctor API, a project to analyze/monitor the OSS health status

This is the first project I did after I joined Dell.

**Comment by andyzhouty** [comment]
- Author: [z-t-y](https://github.com/z-t-y)
- Project name & link: [Djask](https://github.com/z-t-y/Djask)
- Description: An enhanced django-like Flask

**Comment by FarmerChillax** [comment]
- Anthor: [Farmer](https://github.com/FarmerChillax)
- Project name: e广科
- Location: Guangdong, China
- Description: e广科微信小程序后端, 一款校园小程序。

使用 **APIFlask** 重构了使用 **Flask** 的[旧版小程序后端](https://github.com/paicha/gxgk-wechat-server)

**Comment by JunzhiCHEN** [comment]
- auth: chenjz 
- Location:  Beijing, China
- Project description:   一个team内部问题反馈小程序后端。后面改成demo后，再补链接

**Comment by data-henrik** [comment]
- auth: [data-henrik](https://github.com/data-henrik)
- Project name/description: API backend as custom extension for an IBM Watson Assistant chatbot, interfacing with database
- Project link: https://github.com/IBM-Cloud/slack-chatbot-database-watson, see [app.py](https://github.com/IBM-Cloud/slack-chatbot-database-watson/blob/master/app.py)

**Comment by ghost** [comment]
- Author: [Jack Zhai](https://github.com/wezhai)
- Organization: KNX
- Location: Shanghai, China
- Project name/description: 元数据管理平台

**Comment by dungvv** [comment]
- Author: [dungvv](https://github.com/dungvv)
- Organization: [Fetek](https://fetek.io)
- Location: Vietnam
- Project name/description: Omni channel sale platform

**Comment by ChriZzn** [comment]
- Author: [ChriZzn](https://github.com/ChriZzn)
- Organization: [snapSEC](https://snapsec.at)
- Location: Austria
- Project name / description: Cyber Security Related Projects

**Comment by Pinto-Katende-Jonathan** [comment]
**Author** : Pinto katende Jonathan
**Organisation** : Private
**Location** : Democratic republic of Congo
**_Project name / description_** : Simple Crud project for student in apiflask/React
**Project link** :  https://github.com/Pinto-Katende-Jonathan/KarlProject/tree/main/server

**Comment by yiksun** [comment]
- Organization: Personal
- Location: Canton, China
- Project name/description: Personal web accounting system
- Project link: https://www.shinnpeanut.xyz:8443/

**Comment by pamelafox** [comment]
- Organization: Microsoft
- Location: US
- Project name/description: Charts API - a template app for Python developers, with built-in testing and deployability to Azure Container Apps + CDN.
- Project link: https://github.com/pamelafox/flask-charts-api-container-app

I also wrote a blog post about testing APIFlask with schemathesis:
https://blog.pamelafox.org/2023/02/testing-apiflask-with-schemathesis.html

**Comment by tkzt** [comment]
- Organization: Qiuying RPA
- Location: CN
- Project name/description: Qiuying Console - A RPA console used to manage processes.
- Project link: [https://github.com/qiuying-rpa/console](https://github.com/qiuying-rpa/console)

**Comment by Briscoooe** [comment]
- Organization: Personal
- Location: Ireland
- Project name/description: Irish Rail REST API. A REST API wrapper around an outdated SOAP/XML API.
- Project link: [https://github.com/Briscoooe/irish-rail-rest-api](https://github.com/Briscoooe/irish-rail-rest-api)

**Comment by meiyujack** [comment]
- Organization: Wuhan Pepsi
- Location: Pepsi Yichang Factory
- Project name/description: This is a website for Wuhan Pepsi's assert organization. The feature is using excel to organize asserts.
- Project link: https://github.com/meiyujack/pepsi_assert

**Comment by Bryanthelol** [comment]
Author: BryantHe
Project name: DDK体重管家
Location: Shanghai, China
Description: DDK体重管家微信小程序后端 API backend

使用 APIFlask 编写了“DDK体重管家”微信小程序后端 API

**Comment by andrewlayer** [comment]
- Author: Layer
- Location: Columbus, Ohio
- Project name: Layer
- Description: Making software platforms easier to use
- Project Link: https://buildwithlayer.github.io/buildwithlayer/blog/dry_with_apiflask

**Comment by Bryanthelol** [comment]
Author: BryantHe
Project name: DDKWOW
Location: Shanghai, China
Description: 游戏魔兽世界 PVP 专精评级排行，探索竞技场中各职业专精的强度表现
Project Link: https://www.ddkwow.cool/wow/

使用 APIFlask 编写了“DDKWOW”网站后端 API

**Comment by FarmerChillax** [comment]
- Organization: XPENG
- Location: Guangzhou, China
- Project name/description: Event Service Platform, a project to handle the Event Bus

This is the first project I did after I joined Xpeng - Autonomous Driving Department.

**Comment by FarmerChillax** [comment]
- Organization: XPENG
- Location: Guangzhou, China
- Project name/description: Lance Namespace service

This is the Lance Namespace Spec impl.
